### PR TITLE
first commit of week5 ps, rewriting week1 lab in R

### PR DESCRIPTION
**Pull request by vaga3461**

### CODE REVIEW

**Comment by vaga3461** [comment]
link for reviewers:

https://mybinder.org/v2/gh/CUB-Computational-Tools/assignment_problem_set/binder-R?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2Fassignment_problem_set%26branch%3Dweek5%26urlpath%3Dlab

**Comment by jolsenvaldez** [comment]
I think your binder link that you've included in this thread goes to a general "assignment_problem_set" binder, just an fyi

File: week5.ipynb [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# now we can create a list of constants\n",
+    "\n",
+    "constants <- list('mass_of_apple' = mass, \n",
```
Comment (jolsenvaldez) [inline week5.ipynb:214]: You may want to include that in addition to creating a list of constants you've also created a new VARIABLE called "constants."

File: week5.ipynb [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# calculate potential energy\n",
+    "\n",
+    "potential_energy = constants[[1]] * constants[[2]] * (constants[[4]] - constants[[3]])\n",
```
Comment (jolsenvaldez) [inline week5.ipynb]: It's interesting to see the calculation done this way in R. I've always done it with a dollar sign like constants$gravitational_acceleration where you call the variable list and select a specific value in that list.

**Review by jolsenvaldez — APPROVED** [review]
Looks good to me!

**Pull request merged**